hugo/content/blog/2025-07-11-mumble-1-5-817.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Mumble 1.5.817
+author: Hartmnt
+date: 2025-07-11
+categories:
+  - Release
+---
+
+Today we release the second patch-release in the Mumble 1.5.x series. This release includes fixes for undesired
+ACL behavior. If you are a server administrator, please continue reading the detailed explanation.
+
+You can download the new version from our [**Downloads page**]({{< relref "/downloads" >}}) or the
+[**GitHub release page**](https://github.com/mumble-voip/mumble/releases/tag/v1.5.817) or from within your Windows client or software package
+management system.
+
+<!--more-->
+
+## Changes to some ACL behavior
+
+**Attention Server Administrators**:
+
+We have fixed a few ACL related behaviors which we have determined to be bugs. This is why we are including them in
+this patch-release. If you have ACLs configured which were relying on that undesired behavior, you want to make sure to fix
+your ACL configuration when updating your server to this release. We assume the number of impacted servers will be minimal and
+adapting the ACLs in case your server is affected will be reasonably easy.
+
+* Previously, both the ``Traverse`` and ``Write`` ACL would be evaluated without taking the "in this channel" and "in sub-channels" context options into account. This is now fixed. You can now deny ``Traverse`` for sub-channels, while the channel you are defining the ACL in is still traversable. ({{< issue 3579 >}})
+* Previously, in addition to normal ACL inheritence rules, the ``Write`` ACL would always be inherited from the parent channel. For consistency, this was removed. To prevent "ungovernable" channels, users which are granted ``Write`` on the root channel, will be able to ``Write`` in any channel of the server no matter what these sub-channels configure. ({{< issue 6512 >}})
```

Comment:
```suggestion
* Previously, in addition to normal ACL inheritance rules, the ``Write`` ACL would always be inherited from the parent channel. For consistency, this was removed. To prevent "ungovernable" channels, users which are granted ``Write`` on the root channel, will be able to ``Write`` in any channel of the server no matter what these sub-channels configure. ({{< issue 6512 >}})
```

```suggestion
* Previously, in addition to normal ACL inheritance rules, the ``Write`` ACL would always be inherited from the parent channel. For consistency, this was removed. To prevent "ungovernable" channels, users who are granted ``Write`` on the root channel will be able to ``Write`` in any channel of the server, no matter what these sub-channels configure. ({{< issue 6512 >}})
```